hack/lib/serverless.bash

```diff
@@ -358,6 +358,10 @@ function teardown_serverless {
   done
   oc delete namespace openshift-serverless --ignore-not-found=true
 
+  if [[ ! $(oc get crd -oname | grep -c 'knative.dev') -eq 0 ]]; then
+    oc get crd -oname | grep 'knative.dev' | xargs oc delete --timeout=60s
+  fi
```

Comment:
Are we expecting stuff to hang/fail if it doesn't work as we expect?

Reply:
If the deletion of CRDs hangs it means there's a problem, but hanging is limited to 60 seconds. I didn't want the whole job to be timing out after 2hrs.

Regarding running shutdown in non-upgrade paths. The serverless-operator Golang tests deploy/undeploy KnativeServing and the operator but they don't delete the CRDs. I'm not aware of other places where we'd be testing that.